src/Trait/BrugerRefTrait.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Trait;
+
+use Doctrine\ORM\Mapping as ORM;
+
+trait BrugerRefTrait
+{
+    #[ORM\Column(length: 255, nullable: true)]
+    private ?string $brugerRefUUIDIdentifikator = null;
```

Comment:
Should this be an `UuidV4`? The name hints that it's some kind of UUID …

Reply:
I don't currently see any gain from converting it into UUID from string, even though the name does hint at it being an UUID.